src/styles/header.scss

```diff
@@ -2,11 +2,33 @@
 @import "_design-system.scss";
 @import "~bulma/sass/utilities/initial-variables.sass";
 
+// This placeholder replaces the header when it detaches to
+// become fixed, preventing jump in page content
+.jf-sticky-phantom {
+  @include mobile {
+    display: none;
+    height: $navbar-height-mobile;
+    background-color: $white;
+
+    &.jf-sticky {
+      display: block;
+    }
+  }
+}
+
```

Comment:
This could be a great place to add a CSS transition that slides in/out the header menu

Reply:
Yeah, I started to attempt that, but had a bit of trouble because it's a change from block position to fixed, rather than an actually move, so a basic transition wasn't working for me. I think there's a way to still do this by setting the initial position to off screen once it becomes fixed and slide it down. But wanted to at least get the basics merged in first in case there are other higher priority things today.